style-dictionary/build-theme.js

```diff
@@ -15,12 +15,41 @@ function build(source, destination) {
     },
   });
 
+  const PALETTE_TYPE_DEFINITION = `import { PaletteOptions } from '@mui/material/styles/createPalette';
+
+declare module '@mui/material/styles/createPalette' {
+  export interface Palette {
+<TERRAWARE_PALETTE_TYPE_DEFINITION>
+  }
+
+  export interface PaletteOptions {
+<TERRAWARE_PALETTE_TYPE_DEFINITION>
+  }
+}`;
+
+const TERRAWARE_PALETTE = `type ThemeDictionary = {
+  [key: string]: any;
+};
+
+const TerrawareTheme: ThemeDictionary = <TERRAWARE_PALETTE>
+
+export default TerrawareTheme;`;
```

Comment:
Is this _**`;**_ and error?

Reply:
Nope,, that's the string that will be written to the output file:

```
export default TerrawareTheme;
```

Reply:
The second ; is to terminate the current string value - which internally contains another ; which will end up in the file.

Reply:
Ok, thanks for the explanation